Windows 10 / Visual Studio 2015 compilation error

Using commit https://github.com/godotengine/godot/commit/30d4a50b424c927b178cabfe30302de1e9d4c836, I get a compilation error when building using Windows 10 64-bit using Visual Studio 2015 (compiling to 32 bits).

```
drivers\jpegd\image_loader_jpegd.cpp(66): error C2065: 'uint': undeclared identifier
drivers\jpegd\image_loader_jpegd.cpp(66): error C2146: syntax error: missing ';' before identifier 'scan_line_len'
drivers\jpegd\image_loader_jpegd.cpp(66): error C2065: 'scan_line_len': undeclared identifier
drivers\jpegd\image_loader_jpegd.cpp(67): error C2065: 'scan_line_len': undeclared identifier
```

--- a/drivers/jpegd/image_loader_jpegd.cpp
+++ b/drivers/jpegd/image_loader_jpegd.cpp
@@ -63,7 +63,7 @@ Error ImageLoaderJPG::load_image(Image *p_image,FileAccess *f) {
 	for (int y = 0; y < image_height; y++)
 	{
 		const jpgd::uint8* pScan_line;
-		uint scan_line_len;
+		jpgd::uint scan_line_len;
 		if (decoder.decode((const void**)&pScan_line, &scan_line_len) != jpgd::JPGD_SUCCESS)
 		{
 			return ERR_FILE_CORRUPT;

